ice40/pack.cc

```diff
@@ -793,13 +797,13 @@ static void pack_special(Context *ctx)
                         packagepin_cell->ports.erase(pll_packagepin_driver.port);
                     }
 
-                    log_info("  constrained '%s' to %s\n", packed->name.c_str(ctx), ctx->getBelName(bel).c_str(ctx));
+                    log_info("Constrained '%s' to %s\n", packed->name.c_str(ctx), ctx->getBelName(bel).c_str(ctx));
```

Comment:
A lot of code in nextpnr uses the convention of log "headers" with capitalized initial letter and ending in `..`, followed by detail messages indented and with lowercase initial letter. This is such a detail message. `Packing special functions..` above is the header. This change breaks this convention. Why do that?

(Same with the other `log_info` changes below. Un-indenting the erros is okay imo.)

Reply:
Hm, that's more or less the style I was trying to achieve, but I noticed that these 'detail' messages had no 'header' to hang off of, apart from the 'Packing specials' one. Let me fix this up.